TranslatorReasonerAPI.yaml

```diff
@@ -190,7 +190,14 @@ components:
               $ref: '#/components/schemas/NodeBinding'
         edge_bindings:
           type: object
-          description: List of QEdge-KEdge bindings.
+          description: >-
+            Dictionary of Input Query Graph to Result Knowledge Graph edge
+            bindings where the dictionary keys are the key identifiers of the
+            Query Graph edges and the associated values of those keys are
+            instances of EdgeBinding schema type (see below). This value is
+            an array of EdgeBinding's since a given query node may resolve to
```

Comment:
In the last sentence, may want to replace "query node" with "query edge", and likewise with "knowledge graph nodes"

Reply:
Thank you @patrickkwang and @CaseyTa, I've push a clarification of the comments for this PR based on your feedback.

Reply:
@RichardBruskiewich There's still two mentions of nodes in the last sentence. 

Also, I don't think the knowledge graph edges that are being bound to the same query graph edge all have to have the same predicate. If the query graph edge doesn't specify the type of the edge, then I believe any KG edge can satisfy that QG edge. Similarly, the KG edge could be different descendants of a QG edge with a specified predicate that's higher in the hierarchy.

Reply:
Understood @CaseyTa . Making the modifications which will be committed shortly.

I have also decided to add the QEdge and Edge documentation task to this PR. I'll work on this after committing the repaired EdgeBinding text.